.pre-commit-hooks.yaml

```
@@ -72,3 +72,20 @@
   language: script
   files: \.go$
   exclude: vendor\/.*$
+
+# ---------------------------------------------------------------------------------------------------------------------
+# GNU Make specific hooks
+# ---------------------------------------------------------------------------------------------------------------------
+
+# Validate targets that are marked as PHONY
+- id: phony-targets
+  name: phony-targets
+  description: Linter for detecting targets marked as PHONY that don't exist.
+  entry: pre_commit_hooks/make/phony-targets.sh
+  language: script
+  files: (Makefile|Makefile.*|.+.mak|.+.mk)$
```

`Makefile` will be matched by  `Makefile.*`?

```suggestion
  files: (Makefile.*|.+.mak|.+.mk)$
```

---

resolved in b474bbb